python/mxnet/gluon/block.py

```diff
@@ -1248,7 +1248,11 @@ def export(self, path, epoch=0, remove_amp_cast=True):
             will be created, where xxxx is the 4 digits epoch number.
         epoch : int
             Epoch number of saved model.
-
+        remove_amp_cast : bool, optional
+            Whether to remove the amp_cast and amp_multicast operators, before exporting the model.
+        allow_extra : bool, optional
+            Whether to save extra parameters whose name not in the result symbol. User can set
+            allow_extra to True to load these parameters with old mxnet.mod.Module.set_params api.
```

Comment:
```suggestion
            Whether to save extra parameters whose names are not in the result symbol.
            User can set allow_extra to True to load these parameters with old mxnet.mod.Module.set_params API.
```

Reply:
You may want to backport this to 1.x as we are dropping module support in 2.0

Reply:
Like @wkcn said, someone would like to save extra parameters. We can drop this argument, but it may be useful in some rarely cases.